lib/item.rb

```diff
@@ -1,3 +1,10 @@
+require 'pry'
 class Item
-
+    attr_reader :name,
+                :price
+    
+    def initialize(info)
+        @name  = info[:name]
+        @price = info[:price]
```

Comment:
Per the interaction pattern, the price should be stripped of the "$" and converted to a string